src/SFA.DAS.Employer.PR.Web/Controllers/Requests/DeclineCreateAccountConfirmationController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Employer.PR.Domain.Interfaces;
+using SFA.DAS.Employer.PR.Web.Infrastructure;
+using SFA.DAS.Employer.PR.Web.Models;
+
+namespace SFA.DAS.Employer.PR.Web.Controllers.Requests;
+
+
+[Route("requests")]
+public class DeclineCreateAccountConfirmationController(IOuterApiClient _outerApiClient) : Controller
+{
+    [Authorize]
+    [HttpGet]
+    [Route("{requestId:guid}/createaccount/decline/confirmed", Name = RouteNames.DeclineCreateAccountConfirmation)]
+    public async Task<IActionResult> Index(Guid requestId, CancellationToken cancellationToken)
+    {
+        var requestResponse = await _outerApiClient.GetRequest(requestId, cancellationToken);
```

Comment:
We can send in provider name in the TempData to this page, that way we do not have to make another api call just for provider name. and if the provider name is not there in the TempData we can navigate the user to "Your providers" home page which covers for coming back to this page via browser back.

Reply:
Agreed, but as we discussed, I will leave this to you